.flake8

```diff
@@ -1,5 +1,6 @@
 [flake8]
 select = E901,E999,F821,F822,F823
+ignore = N806, N812, E266, F405
```

Comment:
What do these check?

Reply:
N812 -- lowercase imported as non lowercase (we have ```import torch.nn.functional as F```)
N806 -- variable in function should be lowercase (we have portions in code where we use uppercase)
E266 -- too many leading '#' for block comment
F405 -- Name may be undefined, or defined from star imports